NuGet.config

```diff
@@ -6,12 +6,6 @@
   </fallbackPackageFolders>
   <packageSources>
     <clear />
-    <!--
-      'src/test/PrepareTestAssets/PrepareTestAssets.proj' generates a NuGet.config file using this
-      one as a template. The following line is a marker to insert the test restore sources.
-    -->
-    <!-- TEST_RESTORE_SOURCES_INSERTION_LINE -->
```

Comment:
The change to stop Darc from removing these should be rolled out, closing PR to generate a fresh one with next update.